timApp/static/scripts/tim/document/viewctrl.ts

```diff
@@ -446,12 +445,17 @@ export class ViewCtrl implements IController {
         this.editingHandler.updateEditBarState();
     }
 
-    get editMenuOnLeft() {
-        return this.editMenuOnLeftStorage.get() ?? false;
+    get editMenuOnLeft(): boolean {
+        return (
+            genericglobals().userPrefs.parmenu_position ==
+            ParMenuHandlePosition.Left
+        );
     }
 
     set editMenuOnLeft(value: boolean) {
-        this.editMenuOnLeftStorage.set(value);
+        genericglobals().userPrefs.parmenu_position = value
+            ? ParMenuHandlePosition.Right
+            : ParMenuHandlePosition.Left;
```

Comment:
Right ja Left varmaan pitäisi olla toisinpäin. Tuo samalla ehkä korjaa huomaamasi "kauneusvirheen" 😄

Reply:
Joo eipähän tuo enää sattunut silmään tuolta...